### ISSUE TITLE
Minimize button not working when VSCode window is not in focus

### ISSUE THREAD

**Issue by ediril** [microsoft/vscode#133395]
- VS Code Version:
```
Version: 1.60.0
Commit: e7d7e9a9348e6a8cc8c03f877d39cb72e5dfb1ff
Date: 2021-09-01T10:54:53.442Z (2 wks ago)
Electron: 13.1.8
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Darwin x64 18.7.0
```
- OS Version: macOS Mojave 10.14.6

Steps to Reproduce:

1. Start VSCode and leave its window up on the desktop
2. Start another application and put its window in focus in front of VSCode. Make sure VSCode window is still visible in the back
3. Click on the minimize button on VSCode window while the second application is still in focus.

<img width="579" alt="Screen Shot 2021-09-19 at 1 51 12 AM" src="https://user-images.githubusercontent.com/5505693/133917056-f0a3f00d-9d61-4302-bbf7-7d25e5f27a01.png">

You'll notice that the other app window is still in focus by looking at the macOS menubar at the top (in the example above, the app in focus is Firefox). Clicking on VSCode's minimize button brings it to the foreground while the other app is still in focus, but VSCode does not respond to the minimize button press. Minimize button works only when VSCode window is in focus.

The other window buttons do work properly even when VSCode is not in focus, only the minimize button does not respond. Also note that this used to work, it broke recently. Not sure exactly which update broke it since I haven't been updating my VSCode for a while, but for example, it worked in version 1.52.1.

**Comment by WhiteMac** [comment]
Same issue for me, with Code 1.6.2, on Mojave 10.14.6. Unsure whether Mojave is an important common denominator here.

**Comment by ediril** [comment]
> Unsure whether Mojave is an important common denominator here.

It's not. I have the same problem on Catalina as well.

**Comment by ediril** [comment]
If it helps, VSCodium 1.61.1 does not have this problem

**Comment by WhiteMac** [comment]
Thank you, it's been fixed since 1.61 for me.

**Comment by ediril** [comment]
Yes I confirmed that this is fixed in 1.61.1. I'm closing the ticket.
